docs/user-wallets/overview.md

```diff
@@ -0,0 +1,80 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Introduction to User Wallets
+
+<StateOfTheNetwork/>
+
+:::info
+Need some `testLPX` test tokens to mint a PKP on Chronicle? Use the faucet: https://faucet.litprotocol.com/
+:::
+
+## Introduction
+
+The Lit Protocol offers two novel ways of managing private keys with [Programmable Key Pairs (PKPs)](./pkps/overview.md) and [Wrapped Keys](./wrapped-keys/overview.md). Using either of these two methods, you can provide your users with a seamless onboarding experience to your decentralized application, all while offering them non-custodial wallets with strict access control.
+
+When used to build a wallet, both PKPs and Wrapped Keys can make onboarding into and interacting with web3 significantly simpler and more secure. You can hook up web2-style authentication (i.e. Google OAuth, Passkeys, etc) to these keys to abstract away the complexities of seed phrases and self-custody. Since both PKPs and Wrapped Keys leverage the decentralized Lit network, you don't have to worry about the inherent risks associated with key custodians, ensuring no one but your user can control their wallet and manage the assets within.
+
+## PKPs vs. Wrapped Keys
+
+Programmable Key Pairs (PKPs) and Wrapped Keys are both solutions offered by Lit for managing private keys, but they serve different purposes and have distinct characteristics.
+
+PKPs are ECDSA public/private key pairs created by the Lit network using Distributed Key Generation (DKG). Each Lit node holds a share, and more than two-thirds of these shares must be collected to execute a given action (i.e. signing a transaction).
+
+Wrapped Keys are private keys that are encrypted and stored using Lit's Wrapped Keys backend service. They can be initialized by [importing](./importing-key.md) or [storing](./storing-wrapped-key-metadata.md) an existing private key, or can be [generated](./generating-wrapped-key.md) within a Lit node's trusted execution environment (TEE), meaning the clear text private key never exists outside of a Lit node's TEE.
+
+### PKPs
+
+- Generated and managed through Distributed Key Generation (DKG) across the Lit network
+- Rely on threshold cryptography and Multi-Party Computation (MPC) for generation and usage
+- Support specific cryptographic curves (e.g. `secp256k1` used by Ethereum)
+- Keys never exist in full on any single node, reducing the risk of compromise
+- Ideal for applications requiring high security on supported blockchains
+
+#### Why Use Them
+
+- Enhanced Security: PKPs leverage MPC and threshold cryptography, providing a higher level of security as the key is never fully reconstructed on a single node
+- Distributed Trust: The key generation and management process is distributed across the network, reducing single points of failure
+- Programmability: PKPs can be controlled by [Lit Actions](../sdk/serverless-signing/overview.md), enabling advanced use cases and key management
```

Comment:
Can't Wrapped Keys technically be used via Lit Actions as well? If so, maybe we take it out of this "Why Use Them" PKP-specific section and just put this line in both the "PKPs" and "Wrapped Keys" bullet point lists as a benefit.